src/locales/en.json

```diff
@@ -282,5 +282,52 @@
     "fileSavedToFilename": "Saved to {filename}",
     "canvas": "canvas",
     "selection": "selection"
+  },
+  "colors": {
+    "ffffff": "White",
+    "f8f9fa": "Gray 0",
+    "f1f3f5": "Gray 1",
+    "fff5f5": "Red 0",
+    "fff0f6": "Pink 0",
+    "f8f0fc": "Grape 0",
+    "f3f0ff": "Violet 0",
+    "edf2ff": "Indigo 0",
+    "e7f5ff": "Blue 0",
```

Comment:
I think we should have a map of `Record<Hex, [Color, Shade]>` so that we can only translate a list of those 10 (or how many colors) instead of translating the whole permutation.

Reply:
sounds good to me 👍🏻

Reply:
Tracked as https://github.com/excalidraw/excalidraw/issues/3887.